tests/e2e-pw/bin/test-env-setup.sh

```diff
@@ -40,6 +34,7 @@ if [ -z "$attributes" ] || [ "$attributes" == "[]" ]; then
     ")
 fi
 
+
```

Comment:
Nit: is double blank line here intentional?

Reply:
Fixed with 21b6eb63c779ab64ea0dd4f78e825ab16a206abc